backend/sublet/views.py

```diff
@@ -62,11 +71,37 @@ class Properties(viewsets.ModelViewSet):
     """
 
     permission_classes = [SubletOwnerPermission | IsSuperUser]
-    serializer_class = SubletSerializer
+
+    def get_serializer_class(self):
+        return SubletSerializerRead if self.action == "retrieve" else SubletSerializer
 
     def get_queryset(self):
         return Sublet.objects.all()
 
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)  # Check if the data is valid
+        instance = serializer.save()  # Create the Sublet
+        instance_serializer = SubletSerializerRead(instance=instance, context={"request": request})
+        return Response(instance_serializer.data, status=status.HTTP_201_CREATED)
+
+    def update(self, request, *args, **kwargs):
+        partial = kwargs.pop("partial", False)
```

Comment:
What is `partial` here? Is this some internal django field?

Reply:
I think it's used by the serializer to differentiate patch and put? Just a guess but that makes the most sense.

Reply:
ah, makes sense